behavior.nls

```diff
@@ -85,6 +85,74 @@ to update-landscape
   ]
 end
 
+to update-memories-only 	; procedure to be performed before every movement by the respective agent so that she always remembers the arguments she saw
+    let myx xcor
+    let myy ycor
+    let cur-argum one-of turtles with [(breed = starts or breed = arguments)
+      and xcor = myx and ycor = myy]
+    let add-cur (sentence cur-argum [color] of cur-argum)
+    set subjective-arguments lput add-cur subjective-arguments
+    let to-add []
```

Comment:
For local variables we should develop the convention to comment on what they are doing.